frontend/src/scenes/paths/NewPaths.tsx

```diff
@@ -100,29 +147,28 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
         renderPaths()
     }, [paths, !pathsLoading, size, color])
 
-    function renderPaths() {
+    function renderPaths(): void {
         const elements = document
```

Comment:
Nit: Is there a way to perform a protective check here for the existence of `document`, `canvas.current`, and `elements` here so that no rendering happens as long as any of those don't exist? This might let you drop those `?`'s